src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'StyleLint';
```

Comment:
I think it's lowercase L, isn't it?

Reply:
6ba0ca6

Reply:
cool, casing needs fixing also in PR title + description